website/docs/events/m2.md

```diff
@@ -0,0 +1,127 @@
+
+### Purpose of student submissions
+
+An extensive discussion was undertaken on the nature of student submissions in the SmileyTutor and in OpenEdu.
+
+Options in SmileyTutor have included student submission of material for a very long time.
+The submitted material is typically in the form of examples or drill items.
+The material is typically peer-reviewed by other students and may also be reviewed by a vetted reviewer (usually the instructor).
+
+All of the above is used extensively in the CCAS course.
+
+As assignments, authoring and submitting new material such as drills and examples is fundamentally different from solving existing drills.
+The most common use of material submission is solely to establish a grade in a course.
+However, alternate uses exist and have been implemented in conjunction with SmileyTutor.
+One approach is the use of peer-review as a teaching tool, as the act of grading can give a student important insight not covered by traditional problem-solving.
+
+A considerable and very useful discussion ensued on whether students are good "test designers".
+It should be noted that the SmileyTutor drills are for practice and not intended for exams or tests, but participants’ experience of grade inflation in Moodle peer reviews is also applicable in the OpenEdu project as well as in the SmileyTutor.
+
+This general discussion on the nature or quality of student submissions was quite useful.
+Examples and research results on the lack of quality and general usefulness of student submissions abound.
+A typical example of a student-authored drill in an introductory calculus course is `What is 1+1?`.
+Another example is a common “Adequate” or even “Good” grade for a “Quality of descriptive graph” when no graph was in fact submitted.
+
+Methods in SmileyTutor have been implemented to force students to use the full grading scale for peer-review, to fight grade inflation and to avoid some of these issues.
+
+In spite of all of this, a large number of submissions which receive high peer-grades are not of the quality to be acceptable as a part of course material in any sense.
+
+To counter the last of these issues, submissions which have received a good peer-review (possibly after resubmission) may be subjected to a review by a vetted reviewer (the instructor) in SmileyTutor.
+An exercise or drill which has been deemed of high quality following these reviews can be considered for inclusion in current or future course material.
+
+These multiple uses of authoring, peer-review and vetted review are not seen as fundamental or obvious parts of OpenEdu.
+
+However, OpenEdu has at its core a storage mechanism which can be used to store vetted student-submissions into SmileyTutor.
+This leads to the second important link between the two systems: Storing SmileyTutor student-submissions in OpenEdu.
```

Comment:
Also, I don't think that submissions are going to be stored on GitHub, are they?

Reply:
That too, but it may depend on what you mean by "submissions". If they're like assignment solutions and quiz answers, then no, of course, but contributions such as fixes and improvements may also be regarded as submissions somehow. The term is just vague.